superduperdb/ext/openai/model.py

```diff
@@ -131,7 +131,7 @@ class OpenAIChatCompletion(OpenAI):
     takes_context: bool = True
 
     #: The prompt to use to seed the response.
-    prompt: t.Optional[str] = None
+    prompt: str = ''
```

Comment:
In my conception of these things, `None` means no prompt, use the default, and `''` means set the prompt to be zero length, which are different outcomes?

Reply:
I changed this for two reasons.

1. Our `format` function won't work with `None`.
2. The OpenAI SDK does not deal with `None` correctly in many instances - https://github.com/openai/openai-python/issues/538